torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4576,9 +4576,6 @@ def shape(size, rank, with_batch_channel=True):
         yield SampleInput(make_arg(shape(D, rank)), scale_factor=1.7)
         yield SampleInput(make_arg(shape(D, rank)), scale_factor=0.6)
 
-def reference_inputs_upsample(mode, self, device, dtype, requires_grad, **kwargs):
```

Comment:
The sizes of the inputs are too large. You'd need to make them smaller / keep the reference input as-is and add a small sample input that tests this path.